bgpd/bgp_attr.c

```diff
@@ -268,17 +268,8 @@ void bgp_attr_flush_encap(struct attr *attr)
 
 static bool labels_same(const struct attr *attr1, const struct attr *attr2)
 {
-	uint32_t i;
-
-	if (attr1->num_labels != attr2->num_labels)
-		return false;
-	if (!attr1->num_labels)
-		return true;
-	for (i = 0; i < attr1->num_labels; i++) {
-		if (attr1->label_tbl[i] != attr2->label_tbl[i])
-			return false;
-	}
-	return true;
```

Comment:
You remove the code that you added in "bgpd: add label_tbl and num_labels in attr structure"

Merge commit "bgpd: normalize bgp labels comparison" into "bgpd: add label_tbl and num_labels in attr structure"